### PR TITLE
HDFS-14962. RBF: ConnectionPool#newConnection() error log wrong protocol class

### PR DESCRIPTION
**Pull request by Cosss7**
## NOTICE

Please create an issue in ASF JIRA before opening a pull request,
and you need to set the title of the pull request which starts with
the corresponding JIRA issue number. (e.g. HADOOP-XXXXX. Fix a typo in YYY.)
For more details, please see https://cwiki.apache.org/confluence/display/HADOOP/How+To+Contribute

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 95 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1295 | trunk passed |
| +1 | compile | 32 | trunk passed |
| +1 | checkstyle | 22 | trunk passed |
| +1 | mvnsite | 35 | trunk passed |
| +1 | shadedclient | 886 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 51 | trunk passed |
| 0 | spotbugs | 72 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 71 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 29 | the patch passed |
| +1 | compile | 26 | the patch passed |
| +1 | javac | 26 | the patch passed |
| +1 | checkstyle | 15 | the patch passed |
| +1 | mvnsite | 30 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 887 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 47 | the patch passed |
| +1 | findbugs | 74 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 473 | hadoop-hdfs-rbf in the patch passed. |
| +1 | asflicense | 30 | The patch does not generate ASF License warnings. |
| | | 4221 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1699 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 24d59e1c84ea 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / dd90025 |
| Default Java | 1.8.0_222 |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/1/testReport/ |
| Max. process+thread count | 2496 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-rbf U: hadoop-hdfs-project/hadoop-hdfs-rbf |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/1/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/ConnectionPool.java [hunk 1]
```diff
@@ -374,7 +374,7 @@ public ConnectionContext newConnection() throws IOException {
       throws IOException {
     if (!PROTO_MAP.containsKey(proto)) {
       String msg = "Unsupported protocol for connection to NameNode: "
-          + ((proto != null) ? proto.getClass().getName() : "null");
+          + ((proto != null) ? proto.getName() : "null");
```
Comment (goiri) [inline hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/ConnectionPool.java]: Let's add a unit test.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 94 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1264 | trunk passed |
| +1 | compile | 34 | trunk passed |
| +1 | checkstyle | 24 | trunk passed |
| +1 | mvnsite | 40 | trunk passed |
| +1 | shadedclient | 902 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 54 | trunk passed |
| 0 | spotbugs | 73 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 72 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 31 | the patch passed |
| +1 | compile | 27 | the patch passed |
| +1 | javac | 27 | the patch passed |
| -0 | checkstyle | 16 | hadoop-hdfs-project/hadoop-hdfs-rbf: The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0) |
| +1 | mvnsite | 29 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 868 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 45 | the patch passed |
| +1 | findbugs | 77 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 515 | hadoop-hdfs-rbf in the patch failed. |
| +1 | asflicense | 30 | The patch does not generate ASF License warnings. |
| | | 4243 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdfs.server.federation.router.TestRouterFaultTolerant |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1699 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux b68bfb1c772c 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 42fc888 |
| Default Java | 1.8.0_222 |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/2/artifact/out/diff-checkstyle-hadoop-hdfs-project_hadoop-hdfs-rbf.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/2/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-rbf.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/2/testReport/ |
| Max. process+thread count | 2897 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-rbf U: hadoop-hdfs-project/hadoop-hdfs-rbf |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/2/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java [hunk 2]
```diff
@@ -301,4 +301,14 @@ public void testConfigureConnectionActiveRatio() throws IOException {
 
     tmpConnManager.close();
   }
+
+  @Test
+  public void testUnsupportedProtoExceptionMsg() throws IOException {
+    exceptionRule.expect(IllegalStateException.class);
+    exceptionRule.expectMessage(UnsupportedProto.class.getName());
```
Comment (goiri) [inline hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java]: Can we put the full string here?
BTW, what was the old message exactly?

Comment (Cosss7) [reply hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java]: java.lang.Class

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 88 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1265 | trunk passed |
| +1 | compile | 33 | trunk passed |
| +1 | checkstyle | 23 | trunk passed |
| +1 | mvnsite | 37 | trunk passed |
| +1 | shadedclient | 976 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 52 | trunk passed |
| 0 | spotbugs | 76 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 73 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 32 | the patch passed |
| +1 | compile | 28 | the patch passed |
| +1 | javac | 28 | the patch passed |
| +1 | checkstyle | 16 | the patch passed |
| +1 | mvnsite | 30 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 891 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 49 | the patch passed |
| +1 | findbugs | 72 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 487 | hadoop-hdfs-rbf in the patch failed. |
| +1 | asflicense | 30 | The patch does not generate ASF License warnings. |
| | | 4308 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdfs.server.federation.metrics.TestMetricsBase |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1699 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 7f2183331438 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 42fc888 |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/3/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-rbf.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/3/testReport/ |
| Max. process+thread count | 2663 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-rbf U: hadoop-hdfs-project/hadoop-hdfs-rbf |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/3/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 80 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1210 | trunk passed |
| +1 | compile | 32 | trunk passed |
| +1 | checkstyle | 20 | trunk passed |
| +1 | mvnsite | 39 | trunk passed |
| +1 | shadedclient | 950 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 51 | trunk passed |
| 0 | spotbugs | 69 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 68 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 29 | the patch passed |
| +1 | compile | 27 | the patch passed |
| +1 | javac | 27 | the patch passed |
| +1 | checkstyle | 16 | the patch passed |
| +1 | mvnsite | 29 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 898 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 47 | the patch passed |
| +1 | findbugs | 71 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 478 | hadoop-hdfs-rbf in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 4200 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdfs.server.federation.metrics.TestMetricsBase |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1699 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux fac68eba793b 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 42fc888 |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/4/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-rbf.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/4/testReport/ |
| Max. process+thread count | 2415 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-rbf U: hadoop-hdfs-project/hadoop-hdfs-rbf |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/4/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java [hunk 3]
```diff
@@ -301,4 +301,16 @@ public void testConfigureConnectionActiveRatio() throws IOException {
...
+  public void testUnsupportedProtoExceptionMsg() throws IOException {
+    exceptionRule.expect(IllegalStateException.class);
+    exceptionRule
+        .expectMessage("Unsupported protocol for connection to NameNode: "
+            + UnsupportedProto.class.getName());
+    ConnectionPool.newConnection(conf, TEST_NN_ADDRESS, TEST_USER1,
+        UnsupportedProto.class);
+  }
+
+  interface UnsupportedProto { }
```
Comment (ayushtkn) [inline hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java]: Instead Can you LambdaTestUtils, Like this :
` @Test
  public void testUnsupportedProtoExceptionMsg() throws Exception {
    LambdaTestUtils.intercept(IllegalStateException.class,
        "Unsupported protocol for connection to NameNode: "
            + TestConnectionManager.class.getName(),
        () -> ConnectionPool.newConnection(conf, TEST_NN_ADDRESS, TEST_USER1,
            TestConnectionManager.class));
  }`

Comment (Cosss7) [reply hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java]: Got it.

File: hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java [hunk 4]
```diff
@@ -301,4 +302,15 @@ public void testConfigureConnectionActiveRatio() throws IOException {
...
+  @Test
+  public void testUnsupportedProtoExceptionMsg() throws Exception {
+    LambdaTestUtils.intercept(IllegalStateException.class,
+        "Unsupported protocol for connection to NameNode: "
+            + TestConnectionManager.class.getName(),
+        () -> ConnectionPool.newConnection(conf, TEST_NN_ADDRESS, TEST_USER1,
+            TestConnectionManager.class));
+  }
+
+  interface UnsupportedProto { }
```
Comment (ayushtkn) [inline hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java]: This line now is not required interface UnsupportedProto { } we can remove it.
Apart LGTM

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 81 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1214 | trunk passed |
| +1 | compile | 30 | trunk passed |
| +1 | checkstyle | 20 | trunk passed |
| +1 | mvnsite | 36 | trunk passed |
| +1 | shadedclient | 871 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 48 | trunk passed |
| 0 | spotbugs | 67 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 66 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 30 | the patch passed |
| +1 | compile | 25 | the patch passed |
| +1 | javac | 25 | the patch passed |
| +1 | checkstyle | 15 | the patch passed |
| +1 | mvnsite | 29 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 888 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 45 | the patch passed |
| +1 | findbugs | 69 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 474 | hadoop-hdfs-rbf in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 4104 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdfs.server.federation.router.TestRouterFaultTolerant |
|   | hadoop.hdfs.server.federation.metrics.TestMetricsBase |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1699 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 637bd28ffb25 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 42fc888 |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/5/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-rbf.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/5/testReport/ |
| Max. process+thread count | 2587 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-rbf U: hadoop-hdfs-project/hadoop-hdfs-rbf |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/5/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 1727 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1242 | trunk passed |
| +1 | compile | 31 | trunk passed |
| +1 | checkstyle | 21 | trunk passed |
| +1 | mvnsite | 34 | trunk passed |
| +1 | shadedclient | 862 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 49 | trunk passed |
| 0 | spotbugs | 68 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 65 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 29 | the patch passed |
| +1 | compile | 26 | the patch passed |
| +1 | javac | 26 | the patch passed |
| +1 | checkstyle | 15 | the patch passed |
| +1 | mvnsite | 29 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 875 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 47 | the patch passed |
| +1 | findbugs | 71 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 417 | hadoop-hdfs-rbf in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 5695 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdfs.server.federation.metrics.TestMetricsBase |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/6/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1699 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 2b02d17f76ce 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 320008b |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/6/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-rbf.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/6/testReport/ |
| Max. process+thread count | 2435 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-rbf U: hadoop-hdfs-project/hadoop-hdfs-rbf |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1699/6/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**